renovate.json

```diff
@@ -9,7 +9,7 @@
   "minor": {
     "automerge": false
   },
-  "rangeStrategy": "pin",
+  "rangeStrategy": "bump",
```

Comment:
is it ok that this affects all packages, not just the `cli`?

Reply:
I think it will apply to all packages unfortunately

Reply:
I mean, we'll see if we agree with this strategy after using it for a while basically. 

It will **not** update any of the packages outright due to how the yarn lock works. We still have to manually open PRs for renovate to bump and install the packages, so it will not change the package versions, but instead change the strategy for the auto-PRs that are opened + not enforce pinning when a dev adds a dep.